README.md

```diff
@@ -34,24 +168,17 @@ scrape_configs:
 
   # The job name is added as a label `job=<job_name>` to any timeseries scraped from this config.
   - job_name: 'maia'
-    metrics_path: "/api/v1/metrics"
+    metrics_path: "/federate"
     basic_auth:
       # Corresponds to your OpenStack User and Project
-      username: <user_id>@<project_id>
+      username: <user_id>|<project_id>
```

Comment:
In windows we have domain/user or domain\user. We might use that one.

`ccadmin/d042680@monsoon3/dns_test`

or 

`ccadmin/d042680|monsson3/dns_test`